packages/angular/projects/ui-angular/src/lib/components/authenticator/components/authenticator/authenticator.component.ts

```diff
@@ -29,7 +29,7 @@ export class AuthenticatorComponent implements OnInit, AfterContentInit {
   @Input() services: AuthenticatorMachineOptions['services'];
   @Input() signUpAttributes: AuthenticatorMachineOptions['signUpAttributes'];
   @Input() socialProviders: SocialProvider[];
-  @Input() variation: 'modal' | undefined;
+  @Input() variation: 'default' | 'modal';
```

Comment:
This is great, `default` and `modal` make sense

Reply:
In the future, we may even go so far as `inline`, `ghost`, `plain`, or something if the card styles are too problematic to remove via CSS.